test/gurka/test_phonemes.cc

```diff
@@ -377,25 +377,24 @@ TEST(Standalone, PhonemesWithAltandDirection) {
     std::vector<SignInfo> signs = tile->GetSigns(node_id.id(), pronunciations, true);
 
     uint32_t sign_index = 0;
-    EXPECT_NE(signs.size(), 0);
+    ASSERT_EQ(signs.size(), 1);
```

Comment:
The tests were modified by running it against pre-refactored code and capturing expected values. Then the same test with stricter checks was run against refactored code to ensure all tests passed.